blog-service/2023-03-02-dashboard-migration.md

```diff
@@ -12,4 +12,4 @@ authors:
 
 Beta - To prepare for Dashboards (Classic) deprecation in late 2023, we've released a tool to assist you with migrating from Dashboards (Classic) to Dashboards (New). You can access this tool directly from a Classic dashboard page by clicking on the **Migrate to new dashboards** link at the top of the page.
 
-For more information, including the current limitations of the tool, see [Migrate Dashboards](/docs/dashboards-new/migrate-to-new-dashboards). Start planning and migrating your Classic dashboards now. You can learn more from [our Community page](https://support.sumologic.com/hc/en-us/articles/12796506964247-All-aboard-Moving-forward-with-Sumo-Logic-s-new-dashboards).
+For more information, including the current limitations of the tool, see [Migrate Dashboards](/docs/dashboards-new/dashboards-migration). Start planning and migrating your Classic dashboards now. You can learn more from [our Community page](https://support.sumologic.com/hc/en-us/articles/12796506964247-All-aboard-Moving-forward-with-Sumo-Logic-s-new-dashboards).
```

Comment:
Because you changed the link, it's equivalent to deleting a doc. please follow steps 1 and 2 [here](https://help.sumologic.com/docs/contributing/remove-doc/).

Reply:
Because you deleted `/docs/dashboards-new/migrate-to-new-dashboards`, we need to redirect users to the equivalent doc, `/docs/dashboards-new/dashboards-migration`. So you just need to add that to cid-redirects.json.